implementations/rust/ockam/ockam_command/src/kafka/inlet/create.rs

```diff
@@ -101,19 +102,16 @@ impl Command for CreateCommand {
         let brokers_port_range = self
             .brokers_port_range
             .unwrap_or_else(|| make_brokers_port_range(&self.from));
-        if (brokers_port_range.end() - brokers_port_range.start()) < 100 {
-            opts.terminal.write_line(fmt_warn!(
-                "The brokers ports range has been trimmed down to '{}-{}' because \
-            the max port number {} was exceeded",
-                brokers_port_range.start(),
-                brokers_port_range.end(),
-                u16::MAX
-            ))?;
-            opts.terminal.write_line(fmt_log!(
-                "Consider changing the bootstrap server \
-                port to a lower value that has at least 100 free ports after it\n"
-            ))?;
+
+        // The bootstrap port can't overlap with the brokers port range
+        if self.from.port() >= brokers_port_range.start()
+            && self.from.port() <= brokers_port_range.end()
+        {
+            return Err(
+                miette!("The bootstrap port can't overlap with the brokers port range").into(),
```

Comment:
This could be slightly improved by showing the actual values.

Reply:
Done, now it will read like:

```
The bootstrap port 63442 can't overlap with the brokers port range 63442-63445
```